src/main/java/uk/gov/di/gpg45engine/services/impl/Gpg45ServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.di.gpg45engine.services.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.di.gpg45engine.domain.data.IdentityEvidence;
+import uk.gov.di.gpg45engine.domain.data.IdentityVerificationBundle;
+import uk.gov.di.gpg45engine.domain.dto.CalculateResponseDto;
+import uk.gov.di.gpg45engine.domain.gpg45.EvidenceScore;
+import uk.gov.di.gpg45engine.services.Gpg45Service;
+import uk.gov.di.gpg45engine.services.IdentityEvidenceService;
+import uk.gov.di.gpg45engine.services.ProfileMatchingService;
+
+import java.util.*;
+
+@Service
+public class Gpg45ServiceImpl implements Gpg45Service {
+
+    private final IdentityEvidenceService identityEvidenceService;
+    private final ProfileMatchingService profileMatchingService;
+
+    @Autowired
+    public Gpg45ServiceImpl(
+        IdentityEvidenceService identityEvidenceService,
+        ProfileMatchingService profileMatchingService
+    ) {
+        this.identityEvidenceService = identityEvidenceService;
+        this.profileMatchingService = profileMatchingService;
+    }
+
+    @Override
+    public CalculateResponseDto calculate(IdentityVerificationBundle bundle) {
+        mapIdentityEvidenceToScore(bundle.getIdentityEvidence());
```

Comment:
Maybe this should map the bundle to an object that just contains the scores that are needed by the matching service. this will make the matching service less coupled to the structure of the IdentityVerificationBundle and make adding tests easier